Formula/gobject-introspection.rb

```diff
@@ -42,6 +42,14 @@ class GobjectIntrospection < Formula
     sha256 "740c9fba499b1491689b0b1216f9e693e5cb35c9a8565df4314341122ce12f81"
   end
 
+  # Fix compatibility with PyInstaller on Monterey.
+  # See: https://github.com/pyinstaller/pyinstaller/issues/6354
+  #      https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/303
+  patch do
+    url "https://gitlab.gnome.org/rokm/gobject-introspection/-/commit/56df7b0f007fe260b2bd26ef9cc331ad73022700.diff"
```

Comment:
Any reason we're not using this?
```suggestion
    url "https://gitlab.gnome.org/GNOME/gobject-introspection/-/commit/56df7b0f007fe260b2bd26ef9cc331ad73022700.diff"
```

Reply:
Mostly because I'm not fully sure what the commit's life-cycle is like; i.e., if the merge-request branch has to be force pushed due to revision, will the (now orphaned) commit still exist in the repository or not? But if that's no concern, then we can switch the patch' source URL to the one of the upstream repository.

Reply:
> if the merge-request branch has to be force pushed due to revision, will the (now orphaned) commit still exist in the repository or not?

GitHub won't lose the commit. I don't know what GitLab does. I suppose we can keep it this way.